components/forms/Checkbox/Checkbox.tsx

```diff
@@ -23,18 +23,16 @@ export const Checkbox = (
 
   return (
     <div data-testid="checkbox" className={classes}>
-      <div>
-        <input
-          className="gc-input-checkbox__input"
-          id={id}
-          type="checkbox"
-          name={name}
-          ref={inputRef}
-          {...inputProps}
-        />
-      </div>
+      <input
+        className="gc-input-checkbox__input"
+        id={id}
+        type="checkbox"
+        name={name}
+        ref={inputRef}
+        {...inputProps}
+      />
       <label className="gc-checkbox-label" htmlFor={id}>
-        {label}
+        <span>{label}</span>
```

Comment:
Wondering if we could look at the needs of these label `<span>` to see if we need them here and in other places?